src/Apps/Partner/partnerRoutes.tsx

```diff
@@ -275,7 +228,7 @@ export const partnerRoutes: RouteProps[] = [
             )
           }
 
-          return <Component {...props} />
+          return <Component />
```

Comment:
This is interesting, this route makes a little query (to see if there needs to be a redirect if there are no works), and then renders a vanilla React component. Since `props` is a Relay fragment, you actually get an odd/loud Relay runtime error (the `{...props}` is ubiquitous so took me a few to find)